src/Classes/Entities/DinoEntity.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace DinoFinder\Classes\Entities;
+
+class DinoEntity
+{
+    private $id;
+    private $species;
+    private $diet;
+    private $height;
+    private $weight;
+    private $length;
+    private $killerRating;
+    private $intelligence;
+    private $age;
+    private $imageUrl;
+    private $foodImage;
```

Comment:
are we getting anything for this at the moment? I am wondering if it will be picked up in the story code review.

Reply:
food image that is

Reply:
will keep it in as a maybe needed later for now, unless Charlie says to remove